src/main/java/hudson/plugins/sonar/client/WsClient.java

```diff
@@ -90,7 +100,7 @@ public ProjectQualityGate getQualityGateBefore52() throws Exception {
     return null;
   }
 
-  public ProjectQualityGate getQualityGateMeasures() throws Exception {
+  public ProjectQualityGate getQualityGateMeasures(String projectKey) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00112)